tests/std/tests/P0896R4_views_split/test.cpp

```diff
@@ -335,6 +295,26 @@ constexpr bool instantiation_test() {
     instantiator::call<test_range<forward_iterator_tag, Common::yes, CanView::yes, Copyability::move_only>>();
     instantiator::call<test_range<forward_iterator_tag, Common::yes, CanView::yes, Copyability::copyable>>();
 
+    { // ensure we get something contiguous
+        for (string_view sv : "127..0..0..1"sv | views::split(".."sv)) {
+            assert(!sv.empty());
+        }
+    }
+
+    if (!is_constant_evaluated()) { // test the from_chars example
+        auto ip    = "1.2.3.4"sv;
+        auto parts = ip | views::split('.') | views::transform([](span<char const> s) {
+            int i;
+            from_chars(s.data(), s.data() + s.size(), i);
+            return i;

```

Comment:
If something went wrong (e.g. the splitting process produced empty string_views), the `from_chars` error code isn't being checked, and `int i` is uninitialized (and unmodified in the event of failure). I believe that initializing to `0` and checking the `errc` would be the most robust thing to do, and it's only a couple extra lines.